test/gleam/string_test.gleam

```diff
@@ -164,18 +164,54 @@ pub fn trim_test() {
   |> should.equal("hats")
 }
 
-pub fn trim_left_test() {
+pub fn trim_start_test() {
   "  hats  \n"
-  |> string.trim_left
+  |> string.trim_start
   |> should.equal("hats  \n")
 }
 
-pub fn trim_right_test() {
+pub fn trim_start_rtl_test() {
+  "  עברית  "
+  |> string.trim_start
+  |> should.equal("עברית  ")
+}
+
+pub fn trim_end_rtl_test() {
+  "  עברית  "
+  |> string.trim_end
+  |> should.equal("  עברית")
+}
```

Comment:
These tests surprised me, as a native english speaker (I do not speak any RTL languages), but it seems to match Rust's implementation, so I'd be surprised if it was wrong? Happy to be corrected!

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=3c96c56a9a7da1a7c70898000531ba38

Reply:
Presumably it's because the RTL text is only in the middle, surrounded by LTR text.

Reply:
Ah yeah - I've gone looking and not been able to find an example of a test case that has RTL whitespace I can use to test here. Maybe there is and I'm just not finding it